### PR TITLE
@essex => [Display Ad] adds support for pixel tracking in ads

### PR DESCRIPTION
**Pull request by l2succes**
https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROW&modal=detail&selectedIssue=GROW-596

One thing to note, for sizmek we assume the `pixel_tracking_code` is a URL here, so we'll have to make sure to let the team know to only include the url and not the script tags.

### CODE REVIEW

File: src/Components/Publishing/Layouts/StandardLayout.tsx [hunk 1]
```diff
@@ -177,3 +197,7 @@ const FooterContainer = styled.div`
     margin: 0 20px;
   `};
 `
+
+const TrackerImage = styled.img`
+  display: none;
```
Comment (eessex) [inline src/Components/Publishing/Layouts/StandardLayout.tsx]: if you use `display: none;` does it still track?

File: src/Components/Publishing/Layouts/StandardLayout.tsx [hunk 2]
```diff
@@ -40,6 +40,23 @@ export class StandardLayout extends React.Component<
...
+    }
+    const isSizmek = url.indexOf("serving-sys") !== -1
+    if (isSizmek) {
+      url = url.replace(
+        "[timestamp]",
+        String(new Date().getMilliseconds() / 1000)
+      )
+    }
+
+    // handle doubleclick
```
Comment (eessex) [inline src/Components/Publishing/Layouts/StandardLayout.tsx]: assuming this is a TODO, if so would be good to label as such

File: src/Components/Publishing/Layouts/StandardLayout.tsx [hunk 3]
```diff
@@ -40,6 +40,23 @@ export class StandardLayout extends React.Component<
     this.setState({ isTruncated: false })
   }
 
+  renderPixelTracker(unit) {
+    let url = unit.pixel_tracking_code
```
Comment (eessex) [inline src/Components/Publishing/Layouts/StandardLayout.tsx]: could this be `renderPixelTracker({ pixel_tracking_code })`

File: src/Components/Publishing/Display/DisplayPanel.tsx [hunk 4]
```diff
@@ -87,7 +88,7 @@ export class DisplayPanel extends Component<Props, State> {
 
   @trackImpression(() => "panel")
   trackImpression() {
-    // noop
+    // Add 3rd party impression support
```
Comment (eessex) [inline src/Components/Publishing/Display/DisplayPanel.tsx]: do we need to use the `track` library for this?  Looks like it isn't implemented either way, but assumed we would need to inject a script tag as well.

Comment (l2succes) [reply src/Components/Publishing/Display/DisplayPanel.tsx]: Na, I thought we did but we actually don't need this her

File: src/Components/Publishing/Layouts/StandardLayout.tsx (hunk 1 above)
Comment (l2succes) [reply src/Components/Publishing/Layouts/StandardLayout.tsx]: Yeah it still makes the request as long as width and height aren't set to 0

Comment (l2succes) [reply src/Components/Publishing/Layouts/StandardLayout.tsx]: Actually apparently even that is fine https://stackoverflow.com/a/18448053

**Pull request merged**